src/window/window_wrapper/mod.rs

```diff
@@ -162,41 +164,45 @@ impl GlutinWindowWrapper {
 
     pub fn draw_frame(&mut self, dt: f32) {
         let window = self.windowed_context.window();
-        let current_size = window.inner_size();
-        let previous_size = self.saved_inner_size;
-
-        if previous_size != current_size {
-            trace!("Updating grid size: {:#?}", current_size);
-            self.saved_inner_size = current_size;
-            self.handle_new_grid_size(current_size);
-            self.skia_renderer.resize(&self.windowed_context);
-        }
 
         if REDRAW_SCHEDULER.should_draw() || SETTINGS.get::<WindowSettings>().no_idle {
```

Comment:
I moved this code to the upper part of draw_frame, so font loading could be triggered before resizing.